<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportNomorNonDuplicate</class>
 <widget class="QDialog" name="ImportNomorNonDuplicate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>181</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Import CSV Nomor Non-duplicate</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="3">
           <widget class="QPushButton" name="pushButton_pilih_file_csv">
            <property name="text">
             <string>Pilih File </string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="pushButton_jalankan_upload">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Jalankan Upload</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="4">
           <widget class="QCheckBox" name="checkBox_is_using_header">
            <property name="text">
             <string>Gunakan Header CSV (Kompatibilitas dengan output MS Excel)</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="3">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>   header(opsional)
   nomor</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="3">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Valid Format CSV sebagai berikut:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Import File</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QPushButton" name="pushButton_tutup">
            <property name="text">
             <string>Tutup</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QLineEdit" name="lineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
